metalearners/metalearner.py

```diff
@@ -133,6 +135,41 @@ def _validate_n_folds_synchronize(n_folds: dict[str, int]) -> None:
         raise ValueError("Need at least two folds to use synchronization.")
 
 
+def _evaluate_model(
+    cfes: Sequence[CrossFitEstimator],
+    X: Sequence[Matrix],
+    y: Sequence[Vector],
+    scorers: Sequence[str | Callable],
+    model_kind: str,
+    is_oos: bool,
+    is_treatment: bool,
+    oos_method: OosMethod = OVERALL,
+) -> dict[str, float]:
+    """Helper function to evaluate all the models of the same model kind."""
+    prefix = f"{model_kind}_"
+    evaluation_metrics: dict[str, float] = {}
+    for idx, scorer in enumerate(scorers):
+        if isinstance(scorer, str):
+            scorer_str = scorer
+            scorer_call: Callable = get_scorer(scorer)
+        else:
+            scorer_str = f"custom_scorer_{idx}"
+            scorer_call = scorer
+        for i, cfe in enumerate(cfes):
+            if is_treatment:
```

Comment:
I'm not sure I can follow what we're doing here... :/

Reply:
The idea of this `if` is to name the treatment models (which predict the CATE directly) with `"{treatment_variant}_vs_0"` to clearly indicate that it is predicting the CATE. We have to add 1 to `i` as the indexing starts at 0 but the first model is estimating the CATE of variant 1 against 0.